src/integration/common/mediaPlayerEmbed.js

```diff
@@ -0,0 +1,19 @@
+export default () => {
+  describe('Media player embed', () => {
+    describe('A11y', () => {
+      it('I can see the audio player embed', () => {
```

Comment:
Can we just rename this to `I can see the audio player embed` embed as it is used for both live radio and MAPs

Reply:
do you mean `I can see the media player embed`?

Reply:
Oops, yes (copy paste error!). Thanks for that